src/irmin/s.ml

```diff
@@ -636,6 +642,10 @@ module type STORE = sig
 
   type ff_error = [ `No_change | `Rejected | lca_error ]
 
+  type export_error = [ `Disconnected of hash ]
+
+  type closure_error = [ `Empty_max | export_error ]
```

Comment:
It seems odd to me to explicitly define this type in some places and not in others (such as https://github.com/mirage/irmin/pull/908/files#diff-1be34172140401295ad7258b1add6782R2254). Is there a reason for this?

Reply:
I used the same pattern as for the other errors, i.e. `ff_error` and `lca_error`. They are defined in `S` and thus they are not accessible from `Private` either. You can for instance see that `[ No_change ..` is used in some places instead of `ff_error`. 
I'm not happy about this either. Maybe we can refactor all errors in a future PR?